src/include/sof/lib/ams.h

```diff
@@ -24,6 +24,9 @@
 /* Wildcard for module_id and instance_id values */
 #define AMS_ANY_ID 0xFFFF
 
+/* AMS messages */
+extern const struct sof_uuid ams_kpd_msg_uuid; /* Key-phrase detected message*/
```

Comment:
this does not scale, the AMS service should not have to know about 3rd-party IP messages.

Put differently, we are not going to add a new variable every time someone makes use of AMS.